basin3d/core/schema/query.py

```diff
@@ -93,6 +141,11 @@ def __init__(self, **data):
             if field in data and data[field] and isinstance(data[field], str):
                 data[field] = list([data[field]])
 
+        # convert tuple to lists for some fields; the camel case is for Pydantic validation
+        for field in ["monitoring_feature", "monitoringFeature"]:
```

Comment:
Please submit an issue ticket to implement a better approach for camelCase to snake_case conversion. This conversion can be configured more cleanly using Pydantic’s alias generator or related configuration features, rather than manually handling both cases throughout the code. This will help reduce duplication and potential errors. Thank you!

Reply:
#217